lib/cc/engine/analyzers/file_list.rb

```diff
@@ -52,7 +38,7 @@ def excluded_files
         end
 
         def excluded_paths
-          Array(engine_config["exclude_paths"])
+          Array(engine_config.config["exclude_paths"])
```

Comment:
Same thing here re: `engine_config` owning its data, and providing methods to interact with (sanitized) information?